client/lxd_images.go

```diff
@@ -589,7 +590,10 @@ func (r *ProtocolLXD) tryCopyImage(req api.ImagesPost, urls []string) (RemoteOpe
 				alias.Name = entry.Name
 				alias.Target = fingerprint
 
-				r.CreateImageAlias(alias)
+				err := r.CreateImageAlias(alias)
+				if err != nil {
+					logger.Warn("Failed creating image alias", logger.Ctx{"err": err})
```

Comment:
This should probably return an error as the action requested by the user hasn't happened.

Reply:
As this is in a goroutine I couldn't return the error directly. Instead I set the `err` field of the `remoteOperation`.